codegens/nodejs-axios/lib/util.js

```diff
@@ -0,0 +1,121 @@
+
+/**
+ * sanitizes input string by handling escape characters eg: converts '''' to '\'\''
+ * and trim input if required
+ *
+ * @param {String} inputString
+ * @param {Boolean} [trim] - indicates whether to trim string or not
+ * @returns {String}
+ */
+function sanitize (inputString, trim) {
+  if (typeof inputString !== 'string') {
+    return '';
+  }
+  (trim) && (inputString = inputString.trim());
+  return inputString.replace(/\\/g, '\\\\').replace(/'/g, '\\\'').replace(/\n/g, '\\n').replace(/"/g, '\\"');
```

Comment:
there's no need to sanitize both single as well as double quotes. The need for sanitization is that when you receive a single quote inside a single quoted string, it needs to be escaped. For example
`'ab\'cd'`. But if we have a single quoted string, we can place double quotes inside it, e.g. `'ab"cd'`. Thus, we only need to escape a single one from both.

Reply:
done